dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsCache.java

```diff
@@ -0,0 +1,38 @@
+package com.dotcms.experiments.business;
+
+import com.dotcms.experiments.model.Experiment;
+import com.dotmarketing.business.Cachable;
+
+import java.util.List;
+
+/**
+ * Cache for {@link Experiment}
+ *
+ * @author vico
+ */
+public interface ExperimentsCache extends Cachable {
+
+    String RUNNING_EXPERIMENTS_KEY = "RUNNING_EXPERIMENTS";
```

Comment:
@victoralfaro-dotcms since we can store any Experiment here, regardless of the state, let's rename this variable to "CACHED_EXPERIMENTS_KEY"